microsite/src/pages/plugins/plugins.module.scss

```diff
@@ -100,7 +100,7 @@
   }
 
   :global(.dropdown__label) {
-    font-size: 15px;
+    font-size: 14px;
```

Comment:
I wonder if it actually makes sense to use things like `text-overflow` instead and add `...` to the end if something is too big rather than changing the font-size. This isn't really a long term solution as we could add a new category next week which is even longer which then means we need to change this value again.

Reply:
> I wonder if it actually makes sense to use things like `text-overflow` instead and add `...` to the end if something is too big rather than changing the font-size. This isn't really a long term solution as we could add a new category next week which is even longer which then means we need to change this value again.

@benjdlambert you are right ! changes done please review